packages/@aws-cdk/aws-lambda-python/test/bundling.test.ts

```diff
@@ -229,3 +229,19 @@ test('Bundling with custom build args', () => {
     }),
   }));
 });
+
+test('Bundling with custom build arg for `PIP_EXTRA_INDEX_URL`', () => {
+  const entry = path.join(__dirname, 'lambda-handler');
+  const testPypi = 'https://test.pypi.org/simple/';
+  Bundling.bundle({
+    entry: entry,
+    runtime: Runtime.PYTHON_3_7,
+    buildArgs: { PIP_EXTRA_INDEX_URL: testPypi },
+  });
+
+  expect(DockerImage.fromBuild).toHaveBeenCalledWith(expect.stringMatching(path.join(__dirname, '../lib')), expect.objectContaining({
+    buildArgs: expect.objectContaining({
+      PIP_EXTRA_INDEX_URL: testPypi,
+    }),
+  }));
+});
```

Comment:
Added this to pacify the validate-pr CI. Not sure if there's a way to validate env vars in Docker images (the change we are making here).